s-ramp-ide/srampRepositoryBrowser/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,48 @@
+Manifest-Version: 1.0
```

Comment:
I can't remember -- any reason why this would need to be committed?  Isn't it generated at build time?